ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -168,6 +168,8 @@ def __init__(
             self.config.sourceConfig.config
         )
         self.client = get_connection(self.service_connection)
+        # Flag the connection for the test connection
```

Comment:
Should we remove `metadata: OpenMetadata` from the `DashboardServiceSource` class  since it is inherited from `Source`?

Reply:
oh sure yes we can do that!

Reply:
Same for other service sources in case it applies :)